plugins/heptio-authenticator-aws

```diff
@@ -0,0 +1 @@
+repository = https://github.com/neerfri/asdf-heptio-authenticator-aws
```

Comment:
This should be the URL to the Git repository. e.g. https://github.com/neerfri/asdf-heptio-authenticator-aws.git